### ISSUE TITLE
Neural Network Controller for Autonomous Pile Loading Revised. (arXiv:2103.12379v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5047]
Neural Network Controller for Autonomous Pile Loading Revised. (arXiv:2103.12379v1 [cs.RO])<br>
https://ift.tt/3fdisU3<br>
<p>We have recently proposed two pile loading controllers that learn from human demonstrations: a neural network (NNet) [1] and a random forest (RF) controller [2]. In the field experiments the RF controller obtained clearly better success rates. In this work, the previous findings are drastically revised by experimenting summer time trained controllers in winter conditions. The winter experiments revealed a need for additional sensors, more training data, and a controller that can take advantage of these. Therefore, we propose a revised neural controller (NNetV2) which has a more expressive structure and uses a neural attention mechanism to focus on important parts of the sensor and control signals. Using the same data and sensors to train and test the three controllers, NNetV2 achieves better robustness against drastically changing conditions and superior success rate. To the best of our knowledge, this is the first work testing a learning-based controller for a heavy-duty machine in drastically varying outdoor conditions and delivering high success rate in winter, being trained in summer.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
